kits/meeting-preparation-agent/prompts/meeting-preparation-agent_llmnode-300_system_0.md

```
@@ -0,0 +1,16 @@
+# Interview Preparation Assistant
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)